types/react-client.d.ts

```diff
@@ -164,6 +164,46 @@ export interface SessionProviderProps {
    * If set to `0` (default), the session is not polled.
    */
   refetchInterval?: number
+  /**
+   * If a user leaves your application's tab/window and returns to stale data,
+   * the `SessionProvider` automatically requests
+   * a fresh session in the background.
+   * Disable this by setting this to `false`.
+   */
+  refetchOnWindowFocus?: boolean
+  /**
+   * If your application is opened in multiple tabs/windows (of the same browser instance),
+   * every tab/window will maintain its own copy of the local session state;
+   * the session is not stored in shared storage like `localStorage` or `sessionStorage`.
+   * Any update in one tab/window sends a signal to other tabs/windows
+   * to request an update of their own session state.
+   *
+   * Set `broadcast={false}` for no broadcasting,
+   * or see the object options for more granular control.
+   */
+  broadcast?:
+    | {
+        /**
+         * By default, when a session is updated
+         * (eg.: through client-side call of `getSession`,
+         * on window focus, or when a stale session is updated),
+         * a signal is broadcasted to other tabs/windows
+         * to update their copy of the session.
+         *
+         * @note Since `refetchInterval` sets up its own session polling,
+         * that event is never broadcasted.
+         *
+         * Set this option to `false` if you do not need this behaviour.
+         */
+        session?: boolean
+        /**
+         * By default, when you invoke the `signOut` method, this will broadcast
+         * a signal to other tabs/windows, to delete their copy of the session.
+         * Set this option to `false` if you do not need this behaviour.
+         */
+        signOut?: boolean
+      }
+    | false
```

Comment:
As mentioned above I would not support the boolean as well because of ambiguous behaviour